src/undate/converters/cidoc_crm.py

```diff
@@ -0,0 +1,73 @@
+import rdflib
+
+from undate import Undate
+
+#: CIDOC-CRM namespace
+CIDOC_CRM = rdflib.Namespace("http://www.cidoc-crm.org/cidoc-crm/")
+ISMI_DATE_TYPE = rdflib.Namespace(
+    "http://content.mpiwg-berlin.mpg.de/ns/ismi/type/date/"
+)
+ISMI_CALENDAR_TYPE = rdflib.Namespace(
+    "http://content.mpiwg-berlin.mpg.de/ns/ismi/type/calendar/"
+)
+
+
+class TimeSpan(rdflib.resource.Resource):
+    @property
+    def identified_by(self):
+        # by default, rdflib resource value method will return another Resource
+        return self.value(CIDOC_CRM.P1_is_identified_by)
+
+    @property
+    def label(self):
+        # for ISMI records, label is under the crm identifier/appelation
+        # other examples have it directly under the time span as RDFS.label
```

Comment:
The idea is that the appellation represents the date in its original calendar whereas the Time-Span is the processable Gregorian xsd:Date therefore the appellation has the calendar type and its label has the date rendered in e.g. Hijri (the Time-Span label would rather have the date rendered in Gregorian)

Reply:
Thanks for clarifying, I sort of got what you were doing here but not fully.

How would this ideally map to an undate? Would it be two different undate objects? (they should sort the same). Or for this case would you ignore/bypass calendar conversion?